libs/providers/vmware.py

```
@@ -1383,6 +1383,173 @@ def clone_vm(
+        for shared_vmdk_path, positions in shared_vmdks.items():
+            LOGGER.info(f"Processing shared VMDK: {shared_vmdk_path}")
+
+            # First VM's clone is the owner (keeps its independent disk copy)
+            owner_idx, owner_bus, owner_unit = positions[0]
+            owner_clone = cloned_vms[owner_idx]
+
+            owner_disk = self._find_disk_by_scsi_position(owner_clone, owner_bus, owner_unit)
+            if not owner_disk:
+                LOGGER.warning(
+                    f"Could not find shared disk on owner clone '{owner_clone.name}' "
+                    f"at SCSI({owner_bus}:{owner_unit}), skipping"
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)